### PR TITLE
2.x bridging fixes

### PR DESCRIPTION
**Pull request by qiangdavidliu**

### CODE REVIEW

File: eureka2-ext/eureka2-eureka1-rest-api/src/main/java/com/netflix/eureka2/eureka1/rest/query/Eureka2FullFetchWithDeltaView.java [hunk 1]
```diff
@@ -139,19 +140,32 @@ public RegistryFetch apply(List<ChangeNotification<InstanceInfo>> updates) {
 
             for (ChangeNotification<InstanceInfo> update : updates) {
                 com.netflix.appinfo.InstanceInfo v1Update = toEureka1xInstanceInfo(update.getData());
+                if (v1Update== null) {
```
Comment (tbak) [inline eureka2-ext/eureka2-eureka1-rest-api/src/main/java/com/netflix/eureka2/eureka1/rest/query/Eureka2FullFetchWithDeltaView.java]: Is it possible that v1Update is null? It should never happen in properly working app.

File: eureka2-server/build.gradle [hunk 2]
```diff
@@ -23,6 +23,7 @@ dependencies {
     compile "org.slf4j:slf4j-simple:${slf4j_version}"
     compile "commons-cli:commons-cli:${commons_cli_version}"
 
+    compile "org.codehaus.jettison:jettison:${jettison_version}"
```
Comment (tbak) [inline eureka2-server/build.gradle]: why this dependency?

File: eureka2-ext/eureka2-eureka1-rest-api/src/main/java/com/netflix/eureka2/eureka1/rest/query/Eureka2FullFetchWithDeltaView.java (hunk 1 above)
Comment (qiangdavidliu) [inline eureka2-ext/eureka2-eureka1-rest-api/src/main/java/com/netflix/eureka2/eureka1/rest/query/Eureka2FullFetchWithDeltaView.java]: see change to toEureka1xInstanceInfo() which catches exceptions during conversion and returns null.

File: eureka2-server/build.gradle (hunk 2 above)
Comment (qiangdavidliu) [inline eureka2-server/build.gradle]: karyon-admin brings in jettison 1.1, which does not work with serializing v1.Applications. (http://jettison.codehaus.org/)

**Pull request merged**